c/src/lib.rs

```diff
@@ -195,10 +195,22 @@ pub struct FoxgloveServerCallbacks {
             params: *const FoxgloveParameterArray,
         ) -> *mut FoxgloveParameterArray,
     >,
+    pub on_parameters_subscribe: Option<
+        unsafe extern "C" fn(
+            context: *const c_void,
+            param_names: *const *const c_char,
```

Comment:
What about using foxglove string here? It seems inconsistent to use c strings in some places and strings with length elsewhere.